README.md

```diff
@@ -107,10 +106,10 @@ Step 1: Review the Starter Code
 Step 2: Build the Products
 Step 3: Create the Cart Functionality
 
-- `addProductToCart()`
-- `increaseQuantity()`
-- `decreaseQuantity()`
-- `removeProductFromCart()`
+- `addToCart()`
+- `increase()`
```

Comment:
front.js and script.test.js will need to be updated with these new function names

Reply:
I will work on this and make a new pull request.